lib/logstash/outputs/elasticsearch.rb

```diff
@@ -50,114 +49,18 @@
 # Keep in mind that a connection with keepalive enabled will
 # not reevaluate its DNS value while the keepalive is in effect.
 class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
-  attr_reader :client
+  require "logstash/outputs/elasticsearch/http_client"
```

Comment:
For clarity, since this is in the class def it isn't really lazy. You needn't call a method to do the load.